build.gradle

```diff
@@ -160,10 +161,6 @@ publishing {
     }
 
     publications {
-      shadow(MavenPublication) { publication ->
-        project.shadow.component(publication)
-      }
```

Comment:
Apparently this is already done automatically by the release plugin:
https://github.com/johnrengelman/shadow/blob/129549107b6c26ae7000d427c0d60652ab6af788/src/main/groovy/com/github/jengelman/gradle/plugins/shadow/ShadowExtension.groovy#L23-L27